frontend/src/components/user-prs/user-prs-modal.tsx

```
@@ -0,0 +1,381 @@
+  const badgeVariant = getPRStateBadgeVariant(pr.state, pr.merged);
+
+  return (
+    <div
+      className={cn(
+        "cursor-pointer border-b last:border-b-0 transition-colors",
+        isSelected
+          ? "bg-muted border-l-4 border-l-primary"
+          : "hover:bg-muted/50 border-l-4 border-l-transparent"
+      )}
+      onClick={onClick}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)